src/pages/index.tsx

```diff
@@ -32,39 +33,64 @@ const Home: NextPage = () => {
   const { angle, ref } = useElementMouseRelativeAngle();
 
   return (
-    <div className="flex h-screen w-screen flex-col">
-      <Navbar />
-      <div className="flex w-screen grow flex-col items-center justify-center gap-16 p-12 md:flex-row xl:gap-32">
-        <div className="flex w-full flex-col items-center justify-center gap-4 md:w-[300px] md:items-start">
-          <div className="text-5xl font-bold">🪂</div>
-          <div className="text-5xl font-bold">when2meet</div>
-          <div className="text-orange-elevated mt-[-4px] text-6xl font-bold">
-            Elevated
-          </div>
-          <div className="mt-2 text-gray-500">
-            Open-source, easy-to-use, free forever
-          </div>
-        </div>
-        <div
-          ref={ref}
-          className="bg-colorful drop-shadow-pink h-[250px] w-full rounded-[8px] p-[2px] sm:max-w-[350px]"
-          style={
+    <>
+      <NextSeo
+        title="Parachute | An Elevated When2Meet"
+        description="A modern multi-user scheduling web application that is open-source, easy-to-use, and free forever. 
+        Achieved effortless retrieval of all previous activity records, secured login with single sign-on services, 
+        enhanced data privacy, and elevated user experience compared with original when2meet." 
+        openGraph={{
+          url: "https://parachute.fyi",
+          title: "Parachute | An Elevated When2Meet",
+          description:
+            "A modern multi-user scheduling web application that is open-source, easy-to-use, and free forever.",
+          images: [
             {
-              "--colorful-bg-degree": `${angle - 90}deg`,
-            } as React.CSSProperties
-          }
-        >
-          <div className="flex h-full w-full flex-col items-center justify-center gap-4 rounded-[6px] bg-white p-8">
-            <div className="mb-2 text-sm text-gray-500">
-              Try Parachute right now
+              url: "para_og1.png",
+              width: 1200,
+              height: 630,
+              alt: "Parachute",
```

Comment:
We should write better alt messages for our images, these alt tags are for when the imgs can't load to act as a backup text. So we can put something like "Prachute SEO Image" or smt.